.eslintrc

```diff
@@ -1,13 +1,3 @@
 {
-  "extends": "eslint-config-kinvey-platform",
-  "env": {
-    "mocha": true,
-    "node": true
-  },
-  "globals": {
-    "expect": true
```

Comment:
It seems common to define `expect` as a global variable and skip the need to require it in each and every test file. The latter can easily become annoying IMO. What's your take on this?

Reply:
I think it is required there but still, if not declared as global, eslint complains.

Reply:
I cannot see it there. What happens when we set it there - still eslint errors?

Reply:
It requires `common.js` and inside this file `expect` is required.